docs/content/concepts/webserver/ui.mdx

```diff
@@ -35,253 +35,603 @@ Note that several Dagster features, like schedules and sensors, require the Dags
-width={3574}
-height={1962}
+alt="UI Resource details - Uses tab"
+src="/images/concepts/webserver/resource-details-uses-tab.png"
+width={3046}
+height={1618}
+/>
+
+</TabItem>
+</TabGroup>
+</TabItem>
+</TabGroup>
+
+---
+
+## Backfills
+
+- **Description**: The **Backfills** tab contains information about the backfills in your Dagster deployment, across all code locations. It includes information about when the partition was created, its target, status, run status, and more.
+
+- **Accessed by**: Clicking **Overview (top nav) > Backfills tab**
+
+<!-- ![UI Backfills tab](/images/concepts/webserver/backfills-tab.png) -->
+
+<Image
+alt="UI Backfills tab"
+src="/images/concepts/webserver/backfills-tab.png"
+width={3046}
+height={1618}
 />
 
 ---
 
+## Jobs
+
+<TabGroup>
+<TabItem name="All jobs">
+
+### All jobs
+
+- **Description**: The **Jobs** page lists all [jobs](/concepts/ops-jobs-graphs/jobs) defined in your Dagster deployment across all code locations. It includes information about the job's schedule or sensor, its latest run time, and its history. Click a job to open the [**Job details**](#job-details) page.
+
+- **Accessed by**: Clicking **Overview (top nav) > Jobs tab**
+
+<!-- ![UI Job Definition](/images/concepts/webserver/jobs-tab.png) -->
+
+<Image
+alt="UI Job Definition"
+src="/images/concepts/webserver/jobs-tab.png"
+width={3064}
+height={1618}
+/>
+
 </TabItem>
-</TabGroup>
+<TabItem name="Job details">
+
+### Job details
 
-### Schedules and sensors
+- **Description**: The **Job details** page contains detailed information about a job. Click the tabs below for more information about the tabs on this page.
+
+- **Accessed by**: Clicking a job in the [**Jobs** page](#all-jobs)
 
 <TabGroup>
-<TabItem name="Schedules">
+<TabItem name="Overview tab">
 
-#### Schedules
+#### Overview tab
 
-The Schedules page lists all schedules defined in your workspace, as well as information about upcoming ticks for anticipated scheduled runs.
+- **Description**: The **Overview** tab in the **Job details** page shows the graph of assets and/or ops that make up a job.
 
-Click a schedule to view details about the schedule, including its recent tick history and recent runs.
+- **Accessed by:** Clicking the **Overview** tab in the [**Job details**](#job-details) page
 
-<!-- ![UI Schedules page](/images/concepts/webserver/schedules.png) -->
+<!-- ![UI Job Definition](/images/concepts/webserver/jobs-overview-tab.png) -->
 
 <Image
-alt="UI Schedules page"
-src="/images/concepts/webserver/schedules.png"
-width={2315}
-height={1300}
+alt="UI Job Definition"
+src="/images/concepts/webserver/jobs-overview-tab.png"
+width={3066}
+height={1628}
 />
 
----
+</TabItem>
+<TabItem name="Launchpad tab">
+
+#### Launchpad tab
+
+- **Description**: The **Launchpad tab** provides a configuration editor to let you experiment with configuration and launch runs. **Note**: This tab will display only if the job requires config.
```

Comment:
Is this note true? I thought it showed for any op-based job. I could be wrong about that, but we should double check

Reply:
Ooo that may be true - I was looking at assets for this, not realizing the behavior may be different. This did appear to be true for assets, but I'll definitely verify.

Reply:
ok cool - this might be a good q for josh or one of the UI folks, they might know exactly when the launchpad tab shows off the top of their head